Makefile

```diff
@@ -108,7 +111,9 @@ clean:
 		test/copyimg/copyimg \
 		test/testdata/redis-image \
 		$(MANPAGES)
-	$(MAKE) -C contrib/python clean
+ifdef HAS_PYTHON3
+		$(MAKE) -C contrib/python clean
```

Comment:
I didn't see anything Python-3-specific about this on a quick skim.  Can we call this either way, and make any required `setup.py` adjustments needed (possibly none) to get this working with any Python?

Reply:
Well technically we only support py3 ... so that's why I did it that way.

Reply:
> Well technically we only support py3 ... so that's why I did it that way.

Right.  I'm just looking for clean ways around having `:= $(shell ...)` variables in the `Makefile` (#777).  That said, a `command` call won't take *that* long, so if you feel like my "clean way around" isn't clean enough, I'll survive ;).